src/Features/CSharp/Portable/UseExpressionBodyForLambda/UseExpressionBodyForLambdaCodeFixProvider.cs

```diff
@@ -64,7 +65,7 @@ public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
 
             editor.ReplaceNode(
                 declaration,
-                Update(semanticModel, declaration, useExpressionBody));
+                (current, g) => Update(semanticModel, useExpressionBody, declaration, (LambdaExpressionSyntax)current));
```

Comment:
nit: consider naming the last two arguments to `Update`.
(same feedback for use of `Update` in `UseExpressionBodyForLambdaCodeRefactoringProvider.cs`)

Reply:
Done.